MapboxNavigation/RouteVoiceController.swift

```diff
@@ -57,14 +57,18 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     public var bufferBetweenAnnouncements: TimeInterval = 3
     
     /**
+     Delegate used for getting metadata information about a particular spoken instruction.
+     */
+    public weak var voiceControllerDelegate: VoiceControllerDelegate?
+    
+    /**
      Default initializer for `RouteVoiceController`.
      */
     override public init() {
         super.init()
         
         if !Bundle.main.backgroundModes.contains("audio") {
-            print("Voice guidance may not work properly. " +
-                  "Add audio to the UIBackgroundModes key to your app’s Info.plist file")
+            assert(false, "Voice guidance may not work properly. Add audio to the UIBackgroundModes key to your app’s Info.plist file")
```

Comment:
> Voice guidance may not work properly.

With this assertion, voice guidance _definitely_ won’t work properly – the application will crash before voice guidance could ever work. How about:

> This application’s Info.plist file must include “audio” in UIBackgroundModes. This background mode is used for spoken instructions while the application is in the background.